experimental/regression_suite/shark-test-suite-models/sdxl/test_vae.py

```diff
@@ -70,7 +70,7 @@ def SDXL_VAE_COMMON_RUN_FLAGS(
     "--iree-dispatch-creation-enable-aggressive-fusion=true",
     "--iree-codegen-llvmgpu-use-vector-distribution=true",
     "--iree-execution-model=async-external",
-    "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline,iree-preprocessing-pad-to-intrinsics)",
+    "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline,iree-preprocessing-pad-to-intrinsics,util.func(iree-preprocessing-generalize-linalg-matmul-experimental))",
```

Comment:
Can the compiler do this automatically when it is needed / beneficial, without adding yet another flag? We should be removing flags from these tests, not adding them.

Reply:
I agree, I'm just not sure if there is a good way to determine when it would be beneficial. This will always be good for forming dispatches, but it might result in `linalg.generic` contraction shapes that don't have performant code generation. @MaheshRavishankar or @qedawkins might have better insights into this, though.

Reply:
We need to turn it on for AMDGPU backends, but not all other backends handle that ass effectively. The path to get there is a long one. I'd stick with the flag for now (sorry, but there are something that are hard to make work on all backends as well without lot more hands).